livestream/control.go

```diff
@@ -110,6 +114,9 @@ func (ls *Livestreamer) End(ctx context.Context, strm Livestream) error {
 	if err != nil {
 		return fmt.Errorf("failed to update status: %w", err)
 	}
+	if err := ls.CreateEvent(ctx, strm.ID, EventEnded, EventEndedPayload{}); err != nil {
+		log.Printf("failed to log stream end event: %v", err)
```

Comment:
I know it's not mission critical if this fail, but shouldn't the error be handled or passed up?